SwiftCheck/Test.swift

```diff
@@ -6,6 +6,78 @@
 //  Copyright (c) 2015 TypeLift. All rights reserved.
 //
 
+// MARK: - Testing
+
+/// Property Testing is a more static and expressive form of Test-Driven Development that emphasizes
+/// the testability of program properties - A statement or invariant that can be proven to hold when
+/// fed any number of arguments of a particular kind.  It is akin to Fuzz Testing but is made
+/// significantly more power by the primitives in this framework.
+///
+/// A `Property` in SwiftCheck is more than just `true` and `false`, it is a value that is capable
+/// of producing a framework type called `Prop`, which models individual test cases that themselves
+/// are capable of passing or failing "in the small" with a `TestResult`.  This ability is encoded
+/// by a protocol called `Testable` that is adopted by Bool, Property, Prop, and several other
+/// internal framework types.  The thing to notice is the majority of user-facing functions and
+/// combinators in this library return types that conform to `Testable`.  This enables a high level
+/// of composition and enables the testing of incredibly complex properties that would normally be
+/// expressed by stateful, complected unit tests.
```

Comment:
And now the reader is nervous, and possibly has already abandoned us (this is where I would have stopped reading if I didn't like you and SwiftCheck already). "Combinator" is an unfamiliar term and feels very mathy (your reader probably doesn't know they love math yet; they have to be taught that). Composition isn't a selling point unless you already know why it's awesome. And you've just told them this is good for "incredibly complex properties." But I don't think I have incredibly complex properties, so this is probably way too much for me. Let me go find something a little simpler.